Formula/s/semgrep.rb

```diff
@@ -36,12 +36,9 @@ class Semgrep < Formula
   depends_on "gmp"
   depends_on "libev"
   depends_on "pcre"
-  depends_on "pygments"
   depends_on "python-certifi"
-  depends_on "python-click"
-  depends_on "python-packaging"
-  depends_on "python-typing-extensions"
   depends_on "python@3.11"
+  depends_on "sqlite"
```

Comment:
can this be `uses_from_macos from "sqlite"`?

Reply:
This is fixing indirect linkage that's already happening because `sqlite` is in the dep tree.